datasets/penn_treebank/penn_treebank.py

```diff
@@ -0,0 +1,152 @@
+_CITATION = """\
+@article{marcus-etal-1993-building,
+    title = "Building a Large Annotated Corpus of {E}nglish: The {P}enn {T}reebank",
+    author = "Marcus, Mitchell P.  and
+      Santorini, Beatrice  and
+      Marcinkiewicz, Mary Ann",
+    journal = "Computational Linguistics",
+    volume = "19",
+    number = "2",
+    year = "1993",
+    url = "https://www.aclweb.org/anthology/J93-2004",
+    pages = "313--330",
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+This is the Penn Treebank Project: Release 2 CDROM, featuring a million words of 1989 Wall Street Journal material. This corpus has been annotated for part-of-speech (POS) information. In addition, over half of it has been annotated for skeletal syntactic structure.
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://catalog.ldc.upenn.edu/LDC99T42"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = "LDC User Agreement for Non-Members"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://raw.githubusercontent.com/wojzaremba/lstm/master/data/"
+_TRAINING_FILE = "ptb.train.txt"
+_DEV_FILE = "ptb.valid.txt"
+_TEST_FILE = "ptb.test.txt"
+
+
+class PennTreebankConfig(datasets.BuilderConfig):
+    """BuilderConfig for PennTreebank"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig PennTreebank.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(PennTreebankConfig, self).__init__(**kwargs)
+
+
+class PennTreebank(datasets.GeneratorBasedBuilder):
+    """Load the Penn Treebank dataset."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('my_dataset', 'first_domain')
+    # data = datasets.load_dataset('my_dataset', 'second_domain')
+    BUILDER_CONFIGS = [
+        PennTreebankConfig(
+            name="penn_treebank",
+            version=VERSION,
+            description="Load the Penn Treebank dataset",
+        ),
+    ]
+
+    def _info(self):
+        features = datasets.Features({"tokens": datasets.Value("string")})
```

Comment:
this dataset is supposed to be for POS tagging but I see no POS tagging features.

Is this expected ?

Reply:
Hello @lhoestq , currently it outputs the word level tokens from raw text. 
For the pos-tagging task, I was not able to find the pos-tags for this dataset. In practice, the dataset is also used for word level language modelling task. So is this fine?

Reply:
Ok I see !
Can you mention that in the dataset card in this case ? i.e. say that currently the POS tags are missing and it can only be used for word level language modeling

You can also add language modeling to the task field of the YAML tags of the dataset card

Reply:
I will add the feature details of the dataset in the README to make this explicit if it's fine.

Reply:
Thanks, will do these both soon and update the PR! I will also add some more details in the card.